docs/modules/ROOT/nav.adoc

```diff
@@ -1,6 +1,6 @@
 * xref:index.adoc[Overview]
 //* xref:wizard.adoc[Wizard]
-//* xref:extensibility.adoc[Extensibility]
+* xref:components.adoc[Using Components]
```

Comment:
I think this should be a bit more generic, like _Usage_ or _How to use_. I'm thinking we should also add how to use presets here too.